sqlglot/expressions.py

```diff
@@ -6090,6 +6090,10 @@ class PosexplodeOuter(Posexplode, ExplodeOuter):
     pass
 
 
+class PositionalColumn(Expression):
+    arg_types = {"this": True}
```

Comment:
No need for `arg_types` here, you inherit from `Expression`.
```suggestion
    pass
```